### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by AnEmortalKid**
- add developer versions info to contributing
- add community info to contributing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* And thirdly, the code is more what you'd call "guidelines" than actual rules.
+
+> My only request is that everyone is kind and respectful to each other. -remixz
+
+## Table of Contents
+
+[Code of Conduct](#CODE_OF_CONDUCT.md)
+
+
+## Development Tools
```
Comment (Pholey) [inline CONTRIBUTING.md]: can we omit the blank headers until we are ready to fill it with content?

**Comment by Pholey** [comment]
It may be good to add a section about testing here too, i'd really like all PRs to have tests with them if needed

**Comment by AnEmortalKid** [comment]
Thanks I'll give @Pholey some time jic there's any other changes needed.

**Comment by Pholey** [comment]
Looks great, thanks!

**Pull request merged**